index.html

```diff
@@ -41,68 +35,49 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
           <option value="" id="option6">Other</option>
         </select>
       </div>
-
         <div class="data-container2">
         <label for="" id="radio-title">Would you recommend freeCodeCamp to a friend</label>
         <label for="radio1">Definitely</label>
         <input type="radio" value='Definitely'  id="radio1" name='user-recommend'>
-
         <label for="radio2">Maybe</label>
         <input type="radio" value='Maybe'   id="radio2" name='user-recommend'>
-
         <label for="radio3">Not sure</label>
         <input type="radio" value='Not sure'   id="radio3" name='user-recommend'>
       </div>
-
-
       <div class="data-container3">
-
         <p class='check-title'>What would you like to see improved?</p>
         <label for="check1">
         <input type="checkbox" value="Front-end Projects" id="check1">Front-end Projects</label>
-
         <label for="check2">
         <input type="checkbox" value="Back-end-Projects" id="check2">Back-end-Projects</label>
-
         <label for="check3">
         <input type="checkbox" value="Data Visualization" id="check3">Data Visualization</label>
-
         <label for="check4">
         <input type="checkbox" value="Challenges"   id="check4">Challenges</label>
-
         <label for="check5">
         <input type="checkbox"  value="Open Source Community" id="check5">Open Source Community </label>
-
         <label for="check6">
         <input type="checkbox" value="Gitter help rooms" id="check6">Gitter help rooms</label>
-
         <label for="check7">
         <input type="checkbox" value="Videos" id="check7">Videos</label>
-
         <label for="check8">
         <input type="checkbox" value="City Meetups" id="check8">City Meetups</label>
-
         <label for="check9">
         <input type="checkbox" value="Wiki" id="check9">Wiki</label>
-
         <label for="check10">
         <input type="checkbox" value="Forum" id="check10">Forum</label>
-
         <label for="check11">
         <input type="checkbox" value="Additional Courses" id="check11">Additional Courses     </label>
       </div>
-        
         <div class="data-container4">
           <p>Any comments or suggestions?</p>
           <textarea name="" id="text-area" placeholder="Enter your comment here..."></textarea>
         </div>
         <div class="button-container">
           <input type="submit" id="submit" class="button-style">Submit</input>
         </div>
```

Comment:
You have done a great job with your review but there is an issue with this suggestion because I am not sure if the `<a>` tag would improve the performance and also, considering that is a form the correct option would be the `<input>` tag with `type="submit"` or a `<button>` :wink:
But you are delivering your review with polite communication and with an explanation :raised_hands: